src/azurelinux/3.0/net9.0/fpm/amd64/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN tdnf install -y \
         ca-certificates \
         git \
         icu \
+        pigz \
```

Comment:
We don't need it in this image as we only use this image to run fpm to make rpms.

Reply:
```suggestion
```